website/i18n/en.json

```diff
@@ -114,7 +114,7 @@
         "title": "Gesture Responder System"
       },
       "getting-started": {
-        "title": "Getting Started"
+        "title": "Setting up the development environment"
```

Comment:
I think it would be better to fully rename the "old" getting started section to "setting-up-machine" or something in order to avoid confusion moving forward: I find kind of confusing to see the url still saying "getting started" when I click on the setting up the machine.

Reply:
Also, I think that the button on the main page of the website will still link too this page? But I guess the idea would be to change that to point to the new introduction.